nilearn/datasets/utils.py

```diff
@@ -14,10 +14,30 @@
 import tarfile
 import warnings
 import zipfile
+import shutil
+import tempfile
 
 from .._utils.compat import _basestring, cPickle, _urllib, md5_hash
 
 
+class TempDir(object):
```

Comment:
I did not undertsand why you need to add this class ?

Reply:
I removed this function.